src/Components.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Swagger package.
+ *
+ * (c) EXSyst
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace EXSyst\OAS;
+
+use EXSyst\OAS\Collections\Links;
+use EXSyst\OAS\Collections\Schemas;
+
+final class Components extends AbstractObject
+{
+    /** @var Schemas|Schema[]|Reference[] */
```

Comment:
The array notations should be removed here imo since it's not the reality

Reply:
@GuilhemN it's rather hints for IDEs autocomplete and for devs so they don't have to open Spec page, what I think can do is make it consistent across the lib, or yes, just delete them.

Reply:
Will `Schemas` implement `ArrayAccess`? If yes this type hint makes sense indeed.